orttraining/orttraining/core/optimizer/memory_optimizer/recompute_analysis.cc

```diff
@@ -68,15 +68,15 @@ using OpsetToIgnorableIndicesMap = InlinedHashMap<int, IgnorableInputIndices>;
  * 3. Some ops are not supported in older opsets, we need to check whether it is applicable to recompute or not.
  */
 const InlinedHashMap<std::string, OpsetToIgnorableIndicesMap>& GetAllowedRecomputeOps(int probe_op_level) {
+  static std::once_flag is_initialized;
```

Comment:
You don't need to use call_once. You can use a function to initialize recomputable_op_table_map, and C++ standard ensures that the function will only be called once and it is thread-safe.